upgrade-anbox.md

```diff
@@ -8,46 +8,13 @@ The upgrade instructions detail the revisions each charm needs to be upgraded to
 
 ## Upgrade OS
 
-Before you run the upgrade of the appliance or charms below you should make sure all packages on the machines part of the deployment are up-to-date. To do so you can run the following commands on each machine
+Before you run the upgrade of the charms, you should make sure all packages on the machines that are part of the deployment are up-to-date. To do so, run the following commands on each machine:
 
     $ sudo apt update
     $ sudo apt upgrade
 
-## Apppliance based deployments
-
-The Anbox Cloud Appliance includes an `upgrade` command which will perform all relevant upgrade steps to a newer version of the appliance.  First you can check if an update is available:
-
-    $ anbox-cloud-appliance status
-    status: ready
-    update-available: true
-    reboot-needed: false
-
-> **IMPORTANT:** While the upgrade process is active API endpoints and the dashboard will not be available. Anbox containers  will stay active and existing streams will also not be interrupted.
-
-In the command ouput above the `update-available` field indicates an update is available. The upgrade process can now be initiated by running the `upgrade` command:
-
-    $ anbox-cloud-appliance upgrade
-
-The appliance will perform now all necessary steps to upgrade to the newer available version. You can watch for progress on the web interface
-
-![appliance-upgrade|690x435](upload://2mEtGPT2aVrhLvhDW7h9whoEiAT.png) 
-
- or with the `status` command you used above:
-
-    $ anbox-cloud-appliance status
-    status: maintenance
-    progress: 40
-    update-available: false
-    reboot-needed: true
-
-When the upgrade has finished the appliance is again available for regular use.
-
-## Juju based deployments
-
-The following sections describe how to update a Juju based deployment of Anbox Cloud.
-
 <a name="juju-version"></a>
-### Juju Version
+## Check Juju version
 
 With the 1.8 release of Anbox Cloud you **MUST** use Juju >= 2.8. If your deployment doesn't yet use Juju 2.8 you have to upgrade your controller and all models first. See the [Juju documentation](https://juju.is/docs/upgrading-models) for more details on how to upgrade the Juju controller and all models to Juju 2.8.
```

Comment:
```suggestion
With the 1.8 release of Anbox Cloud you **MUST** use Juju 2.8. If your deployment doesn't yet use Juju 2.8 you have to upgrade your controller and all models first. See the [Juju documentation](https://juju.is/docs/upgrading-models) for more details on how to upgrade the Juju controller and all models to Juju 2.8.
```
Since we don't support 2.9 yet, we should tag 2.8